strongbox-data-service/src/main/java/com/orientechnologies/orient/object/jpa/OJPAObjectDatabaseTxEntityManager.java

```diff
@@ -202,12 +207,37 @@ public Query createNativeQuery(String sqlString,
         throw new UnsupportedOperationException("createNativeQuery");
     }
 
+    @Override
+    public StoredProcedureQuery createNamedStoredProcedureQuery(String s) {
```

Comment:
Could you please re-indent you changes to follow the [coding convention](https://github.com/strongbox/strongbox/wiki/Coding-Convention)?

Reply:
Fixed